src/Type/Traits/SubstractableTypeTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Traits;
+
+use PHPStan\Type\SubtractableType;
+use PHPStan\Type\Type;
+use PHPStan\Type\UnionType;
+use PHPStan\Type\VerbosityLevel;
+use function sprintf;
+
+trait SubstractableTypeTrait
+{
+
+	public function describeSubtractedType(?Type $subtractedType, VerbosityLevel $level): string
```

Comment:
I dunno if it's worth to add this method to the interface (I didn't) but since the logic was used 5 times I thought it could been useful to have a specific method (for later updates ?)